app/views/wikies/edit.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1>Editing Wiky</h1>
-
 <%= render 'form' %>
+<%= render 'collaborations/form', collaborations: @collaboration, wiky: @wiky  %><br>
```

Comment:
This is a little confusing, where plural and singular names are mixed: `collaborations: @collaboration`